firstvoices/backend/tests/test_apis/test_wotd_api.py

```diff
@@ -0,0 +1,165 @@
+import json
+
+import factory
+import pytest
+from django.utils.timezone import datetime, timedelta
+from factory.django import DjangoModelFactory
+from rest_framework.test import APIClient
+
+from backend.models.constants import Role, Visibility
+from backend.models.dictionary import WordOfTheDay
+from backend.tests.factories import (
+    DictionaryEntryFactory,
+    MembershipFactory,
+    SiteFactory,
+    get_non_member_user,
+    get_site_with_member,
+)
+from backend.tests.test_apis.base_api_test import BaseSiteControlledContentApiTest
+
+
+class WordOfTheDayFactory(DjangoModelFactory):
+    dictionary_entry = factory.SubFactory(DictionaryEntryFactory)
+    date = datetime.today()
+
+    class Meta:
+        model = WordOfTheDay
+
```

Comment:
OK I know we *just* decided to put factories right in the relevant tests, but could you please put it in the factories.py instead? It will make it work with the new list permission tests. Sorry for the back and forth!

Reply:
No worries. Thank you for adding tests that go through all the models.